### DISCUSSION TITLE
I templated a website design on powerpoint. Please help me turn it into a real website.

### DISCUSSION THREAD

**Discussion by thomaschicago847** [learning-zone/website-templates#46] [General]
I am a musician creating a webpage for myself and don't have a lot of money to spend. I designed a simple website design on powerpoint. I need helping turning it into a real website. I do not want to use no-code platforms like Wix, Squarespace, GoDaddy, etc. I'm not sure I want to use WordPress either.

[WEBSITE.pdf](https://github.com/learning-zone/website-templates/files/12883942/WEBSITE.pdf)

**Comment by Joh19** [comment]
possible contact me via whatsapp +254 702 415215
